src/app/dashboard/hero-card/hero-card.component.html

```diff
@@ -0,0 +1,17 @@
+<div class="hero" *ngIf="hero" [class.active]="isActive" (click)="toggleHero.emit(!isActive)">
+  <img [src]="hero.imageSrc" alt="avatar hero" width="100" height="100">
+  <div class="hero-info">
+    <p class="hero-info-name"><b [title]="hero.name">{{ hero.name }}</b></p>
+    <p><b>Health</b>: {{ hero.health }}</p>
+    <p><b>Armour</b>: {{ hero.armour?.health }}</p>
+    <p><b>Damage</b>: {{ hero.weapon?.damage }}</p>
+    <p>
+      <select (click)="clickWeapon($event)" (change)="changeWeapon($event)" [ngModel]="hero.weapon?.id" class="hero-info-weapon">
+        <option *ngFor="let weapon of weapons" [value]="weapon.id">
+          <!-- <img [src]="weapon.imageSrc" alt="weapon icon"> -->
+          {{ weapon.name }}
+        </option>
+      </select>
+    </p>
+  </div>
+</div>
```

Comment:
I was really impressed when I saw this b/c it showed good intuition on how to do the hero card.  I deliberately didn't wireframe b/c I wanted to find developers who can provide some design that looks good and thinks from the "users experience".

Good job on this.

Remember to add the armour.

Reply:
Thank you!